services/report/__init__.py

```diff
@@ -328,6 +330,24 @@ def create_report_upload(
         flags = normalized_arguments.get("flags")
         flags = flags.split(",") if flags else []
         self._attach_flags_to_upload(upload, flags)
+
```

Comment:
This was done here and not in the base class to only target Coverage type reports